posts/how-to-commit-code-to-a-git-repository-with-visual-studio-code.md

```diff
@@ -0,0 +1,172 @@
+---
+title: How to commit code to a Git repository with Visual Studio Code
+author: marek-zaluski
+tags: [tutorial, git, IDE, open source]
+description: Learn how to create commits in a Git repository with VS Code.
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/how-to-commit-code-to-a-git-repository-with-visual-studio-code.jpg
+imageAlt: Visual Studio Code displaying a diff view
+type: posts
+---
+
+Committing code with Git is useful whether you’re working on your own projects, collaborating with others, or contributing to an open source project. Visual Studio Code is an open source IDE (Integrated Development Environment) with support for many programming languages. In addition to being a capable tool for writing and editing source code, VS Code can help you interact with Git repositories and commit changes.If you’re already using VS Code to edit code, it’s convenient to use the same tool to commit your changes with Git.
+
+In this tutorial, we’ll demonstrate how to use the Git features within VS Code to commit code to a Git repository.
+
+## Prerequisites
+
+To follow this tutorial, you need Visual Studio Code installed on your computer. You can install it by following the instructions on the official [Visual Studio Code website.](https://code.visualstudio.com/)
+
+You also need Git installed. Visual Studio Code’s Git features depend on the Git command-line tools. You can install Git by following the instructions on the [“Installing Git” page of the official documentation.](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+With that software installed, you should have created or identified a Git repository that you would like to contribute to. To make one of your local code repositories a Git repository, you can review the official documentation for “[Getting a Git Repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository).”
+
+## Step 1 — Opening the Source Control panel
```

Comment:
Reading this again, I think this heading can be more descriptive for everything that is happening in this section. Something like, "Step 1 — Preparing to make a commit"